src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -255,50 +305,46 @@ const InvoiceForm = ({
                       >
                         <KeyValue
                           label={<FormattedMessage id="ui-invoice.invoice.billToAddress" />}
-                          value={addressBillTo}
+                          value={addressBillTo || <NoValue />}
```

Comment:
IMHO, this should be leveraged into a separate component, since you display it at least in 2 places - at the form and at the Details pane